main/src/ca/uwaterloo/flix/language/phase/Weeder2.scala

```diff
@@ -1594,6 +1595,17 @@ object Weeder2 {
       }
     }
 
+    private def visitUnsafeExpr(tree: Tree): Validation[Expr, CompilationMessage] = {
```

Comment:
Does this belong in Weeder or Desugarer?

Reply:
Fixed